prow/config.yaml

```diff
@@ -9949,23 +11206,22 @@ periodics:
       secret:
         defaultMode: 256
         secretName: ssh-key-secret
-
-- name: ci-kubernetes-e2e-gke-gpu
-  interval: 2h
+- interval: 6h
```

Comment:
argh, we are now trying to keep prow/config.yaml sorted.

I can split it into two commits.